frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/operator-install-global.spec.ts

```diff
@@ -0,0 +1,130 @@
+  before(() => {
+    cy.login();
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.logout();
+  });
+
+  const operatorPkgName = 'portworx-essentials';
+  const operatorID = 'portworx-essentials-community-operators-openshift-marketplace';
+  const operatorRow = 'Portworx Essentials';
+  const operatorInstallFormURL = `/operatorhub/subscribe?pkg=${operatorPkgName}&catalog=community-operators&catalogNamespace=openshift-marketplace&targetNamespace=`;
+  const operatorInstance = 'StorageCluster';
+  const openshiftOperatorsNS = 'openshift-operators';
+  const operandLink = 'portworx';
+
+  it(`displays subscription creation form for ${operatorName}`, () => {
+    cy.log('navigate to the operator install form from OperatorHub');
+    cy.visit('/operatorhub/all-namespaces');
+    cy.byTestID('search-operatorhub').type(operatorName);
+    cy.byTestID(operatorID).click();
+    cy.log('dismiss Show Community Operator modal');
+    modal.shouldBeOpened();
+    modal.submit();
+    modal.shouldBeClosed();
+    cy.log('go to the install form');
+    cy.byLegacyTestID('operator-install-btn').click({ force: true });
+    cy.url().should('include', operatorInstallFormURL);
+  });
+
+  it(`creates the global install mode ClusterServiceVersion for ${operatorName}`, () => {
+    cy.visit(operatorInstallFormURL);
+    cy.byTestID('install-operator').click();
+    cy.log('verify operator began installation');
+    cy.byTestID('view-installed-operators-btn').should(
+      'contain',
+      `View Installed Operators in namespace ${openshiftOperatorsNS}`,
+    );
+    cy.log('view the ClusterServiceVersion list page');
+    cy.byTestID('view-installed-operators-btn').click();
+    cy.log(`verify the ClusterServiceVersion row for ${operatorRow} exists`);
+    cy.byTestOperatorRow(operatorRow).should('exist');
+  });
+
+  it(`displays details about ${operatorName} ClusterServiceVersion on the "Details" tab`, () => {
+    cy.log(`navigate to the ${operatorName} details page`);
+    cy.byTestOperatorRow(operatorRow).click();
+    cy.byTestSectionHeading('Provided APIs').should('exist');
+    cy.byTestSectionHeading('ClusterServiceVersion Details').should('exist');
+    cy.byLegacyTestID('resource-summary').should('exist');
+  });
+
+  it(`displays empty message on the ${operatorName} ClusterServiceVersion "All Instances" tab`, () => {
+    cy.log('navigate to the "All Instances" tab');
+    cy.byLegacyTestID('horizontal-link-All Instances').click();
+    cy.byTestID('msg-box-title').should('contain', 'No Operands Found');
+    cy.byTestID('msg-box-detail').should(
+      'contain',
+      'Operands are declarative components used to define the behavior of the application.',
+    );
+  });
+
+  it(`displays ${operatorName} ${operatorInstance} creation form`, () => {
+    cy.log('navigate to the form');
+    cy.byLegacyTestID('dropdown-button')
+      .click()
+      .byTestDropDownMenu('storageclusters.core.libopenstorage.org')
```

Comment:
I'm getting concerns about using custom commands in this mannor, noticing some flakes.  This issue may be that by executing `.byTestDropDownMenu` in the middle of the chain, executes `cy.get('[data-test-dropdown-menu="${selector}"]')` -which is not part of the chain, rather starts a new one!   I think we need to make some of our helper commands [Dual-Commands](https://docs.cypress.io/api/cypress-api/custom-commands.html#Dual-Commands). A dual command can either start a chain of commands or be chained off of an existing one.

For this particular case you may just want to use:
```
cy.log('navigate to the form');
    cy.byLegacyTestID('dropdown-button')
      .click()
      .get(`[data-test-dropdown-menu="${selector}"]`)`
```
...until we work out what we need to do.

Reply:
We may need to rewrite our `byXXX` custom selector methods if we want to also use them in middle of a chain (for example):
```
Cypress.Commands.add('byTestDropDownMenu', {
  prevSubject: 'optional'
}, (subject, selector) => {
  // subject may be defined or undefined
  // so you likely want to branch the logic
  // based off of that

  if (subject) {
    // wrap the existing subject
    // and do something with it
    cy.wrap(subject).get(`[data-test-operator-row="${selector}"]`),
  } else {
    cy.get(`[data-test-operator-row="${selector}"]`),
  }
});
```

Reply:
Thanks, @dtaylor113.  Updated per your recommendation.  Will open a PR to make the same change in #7214 .